packages/react-component-library/src/primitives/FloatingBox/partials/StyledArrow.tsx

```diff
@@ -1,18 +1,31 @@
 import styled, { css } from 'styled-components'
-import { Placement } from '@popperjs/core'
+import { BasePlacement } from '@popperjs/core'
 
 interface StyledArrowProps {
-  $placement?: Placement
+  $placement?: BasePlacement
 }
 
 const placementStyleMap = {
```

Comment:
Here there were a few bugs (e.g. the placement `left-start` did not render the arrow correctly) so I've made a few changes beyond the types. Suggest going through the different placement options in the story to compare.